solutions/problem-005/smallestMultiple.py

```diff
@@ -0,0 +1,16 @@
+def smallestMultiple(n):
+	numberToCheck = n
+	found = False
+
+	while not found:
+		if check(numberToCheck, n):
+			found = True
+		numberToCheck += n
+	return numberToCheck - n
+
+def check(numberToCheck, number):
+	for i in range(1, number+1):
+		if numberToCheck%i != 0:
+			return False
+	return True
+print(smallestMultiple(10))
```

Comment:
your code is correct, but please change the number 10 to 20. so I can merge your pull request. thanks